toolkit/tools/imagegen/configuration/parse_partition_test.go

```diff
@@ -14,10 +14,11 @@ var (
 	invalidFstypePartitionCommand = "part / --fstype=     --size=800 --ondisk=/dev/sda"
 	invalidSizePartitionCommand   = "part / --fstype=ext4 --size=abcd --ondisk=/dev/sda"
 
-	validPartitionCommand1 = "part biosboot --fstype=biosboot --size=8 --ondisk=/dev/sda"
-	validPartitionCommand2 = "part / --fstype=ext4 --size=800 --ondisk=/dev/sda"
+	validLegacyBootPartitionCommand = "part biosboot --fstype=biosboot --size=8 --ondisk=/dev/sda"
+	validUEFIBootPartitionCommand   = "part /boot/efi --fstype=efi --size=8 --ondisk=/dev/sda"
```

Comment:
Is the path here identical to `efibootPartition`? If so, we should reuse it here. Same applies to Legacy command.